### ISSUE TITLE
ItemOptionSystem Failed Assertion

### ISSUE THREAD

**Issue by Dumnezeu** [HerculesWS/Hercules#2217]
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Issue Prelude

-  [ ] I have not modified the source prior to reproducing this issue.
-  [ ] I am using the latest version of Hercules.
-  [ ] I am aware that this report will be closed or deleted if it becomes obvious that I am stating the false.

### Description
I downloaded our latest revision of hercules, and i try to test this [http://herc.ws/board/topic/14582-item-option-system-demo-npc/](url) whenever i upgrade to success no error appears and it works fine.. but when the upgrade goes failed this Failed Assertion come out.. [https://imgur.com/M4nLQc1](url) i didn't change any part of the script i use from the link above..

[//]: # (Description of the problem or issue at length.)
[//]: # (Please specify any battle configuration related to the components of this issue that have been changed from the default values. This will allow quicker determination of the cause of the problem.)

**Comment by tlacson7** [comment]
@dastgirp 

its issue with achievement system, not the item option

**Comment by dastgirp** [comment]
Yes, but need more info, as the message does not even show stacktrace
